feat: add option to keep relation scalar fields in json schema output

ericjohney opened the pull request:
I had a need to maintain the relational foreign keys in my json schema.

Would you consider a PR like this to make it optional?

I saw someone else was trying to do something similar in #24

valentinpalkovic commented:
Hi! :) Sorry for the late response. The changes look good!

But please adjust two things:

1. Please squash your commits instead of naming the commit `feature: add option to keep relation scalar fields in json schema output` use `feat: add option to keep relation scalar fields in json schema output` instead, that semantic-release works properly.

2. Please document the introduced option in the README.md. You could add an options table, where all future options and `keepRelationScalarFields` option is documented. Describe the option and it's default value ("false").

ericjohney commented on src/generator/types.ts:
```diff
@@ -11,3 +11,7 @@ export interface ModelMetaData {
 export type PropertyMap = [...DefinitionMap, PropertyMetaData]
+
+export interface TransformOptions {
+    keepRelationScalarFields?: 'true' | 'false'
```
boolean options get parsed as a string from the prisma generator

ericjohney commented on README.md:
````diff
@@ -41,6 +41,21 @@ generator jsonSchema {
+```prisma
+generator jsonSchema {
+  provider = "prisma-json-schema-generator"
+  keepRelationScalarFields = true
````
Do you have any preferences as to whether the option should be documented here as `true` or `"true"`. The prisma generator appears to parse everything as a string anyway.  But I did see another project that used boolean values without the quotes here

valentinpalkovic replied:
I would prefer quotes here. Maybe the team behind Prisma decides to correctly parse boolean values in future releases, which actually would break the plugin usage.

ericjohney replied:
good point! i didn't think of that.

ericjohney commented:
Ok, I fixed the commits (I somehow haven't used semantic-release before, looks cool) and added README.md documentation. Just had a question above about the boolean option formatting.

ericjohney commented:
Thanks @valentinpalkovic! I just updated the documentation to reflect the option as a string value
